src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1403,6 +1403,20 @@ public function testSetBadEnv()
         $this->assertSame('', $process->getErrorOutput());
     }
 
+    public function testEnvBackupDoesNotDeleteExistingVars()
+    {
+        putenv('existing_var=foo');
+        $process = $this->getProcess('echo "$new_test_var"');
```

Comment:
this command does not work on Windows (see appveyor failure)
so we should either make the test pass o Windows, or at least skip it